contrib/credential/osxkeychain/git-credential-osxkeychain.c

```diff
@@ -168,7 +168,7 @@ int main(int argc, const char **argv)
 		"usage: git credential-osxkeychain <get|store|erase>";
 
 	if (!argv[1])
-		die(usage);
+		die("%s", usage);
```

Comment:
Are you working on an upstream fix for this? ;)

Reply:
Yes! Today 😆.